packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -39,7 +35,7 @@ export class JaegerExporter implements SpanExporter {
 
   constructor(config: jaegerTypes.ExporterConfig) {
     const localConfig = Object.assign({}, config);
-    this._logger = localConfig.logger || new NoopLogger();
+    this._logger = localConfig.logger || new api.NoopLogger();
```

Comment:
Suggestion:
This appears to be a common pattern which is effectively necessary boilerplate code.

Provide an api function called getLogger(logger?: Logger) which returns either the provided Logger 
OR a No-op implementation logger.

This allows a couple of things
- We can then don't need ANY NoopLogger class
- No more need for the boilerplate code everywhere that wants to ensure it has a logger instance

> public getLogger(logger?: Logger) {
> if (!logger) {
>    logger = {
>        debug: () => {},
>        error: () => {},
>        warn: () => {},
>        info: () => {},
>    };
>}
>return logger;
>}

Reply:
So this then would (potentially) become
this._logger = api.NoopLogger(localConfig.logger);

Reply:
I think is out of scope for this PR, please create a separate issue "discussion" and move it there. It should not be here, thx